addons/l10n_in_edi/models/account_move.py

```diff
@@ -61,3 +62,16 @@ def _l10n_in_edi_is_managing_invoice_negative_lines_allowed(self):
         """
         param_name = 'l10n_in_edi.manage_invoice_negative_lines'
         return bool(self.env['ir.config_parameter'].sudo().get_param(param_name))
+
+    @api.depends('move_type', 'line_ids.tax_tag_ids', 'amount_residual', 'partner_id.l10n_in_pan')
+    def _compute_l10n_in_edi_display_null_pan_warning(self):
```

Comment:
o_O

you still need to iterate on records in self :warning:

Reply:
oh, I think we will always get a single record. so I removed it.

Reply:
Hello @qdp-odoo ,
I made the changes.